### PR TITLE
add max input controllable via env vars

### LINKED ISSUE #13: limit text input to 500 chars
**description**

We need some kind of limit so people are not allowed to post whole books. We will start with a cap of 500 chars.

### PR DESCRIPTION
**Pull request by JannikStreek**
closes #13

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -23,6 +23,8 @@ services:
       DEFAULT_LOCALE: ${DOCKER_COMPOSE_APP_DEFAULT_LOCALE:-en}
       SECRET_KEY_BASE: ${DOCKER_COMPOSE_APP_SECRET_KEY_BASE:-NDliN2UzNjAzMTlmMWIxODZhNzRjMDIzMDY2OTQ5N2Q4NGU3MjUxZjM0YmI0ZGY1}
       URL_PORT: ${DOCKER_COMPOSE_APP_URL_PORT:-4000}
+      NLP_WORD_TAGGER_MAX_INPUT: 600
+      VITE_NLP_WORD_TAGGER_MAX_INPUT: 600
```
Comment (JannikStreek) [inline docker-compose.yml:27]: VITE needs this prefix. Either we use the VITE variable also inside the server/backend or we need these two.

Comment (nwittstruck) [reply docker-compose.yml:27]: You could set on to the other in the docker-compose.yml. But I'm also fine with keeping it like this.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -14,6 +14,9 @@ name: Create and publish a Docker image
 on:
   release:
     types: [published]
+  push:
+    branches:
+      - 13-add-max-text-input
```
Comment (nwittstruck) [inline .github/workflows/release.yml]: Just remember to delete this afterwards.

**Review by nwittstruck — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (JannikStreek) [reply docker-compose.yml:27]: I think I would keep it like this, as we also need two variables for prod: 1) during the build (Vite for GitHub) and 2) during runtime for the server (elixir)

**Pull request merged**